File: app/src/main/java/com/mutualmobile/praxis/utils/BindingAdapters.java
package com.mutualmobile.praxis.utils;

import android.databinding.BindingAdapter;
import android.view.View;

public class BindingAdapters {

  @BindingAdapter("visibility") public static void setVisibility(View view, boolean visible) {
    view.setVisibility(visible ? View.VISIBLE : View.GONE);
  }
}

File: app/src/main/java/com/mutualmobile/praxis/utils/BindingAdapters.kt
package com.mutualmobile.praxis.utils

import androidx.databinding.BindingAdapter
import android.view.View

object BindingAdapters {

  @BindingAdapter("visibility") fun setVisibility(view: View, visible: Boolean) {
    view.visibility = if (visible) View.VISIBLE else View.GONE
  }
}
